code/datums/chatmessage.dm

```diff
@@ -0,0 +1,87 @@
+#define CHAT_MESSAGE_SPAWN_TIME		1  // 0.1 second
+#define CHAT_MESSAGE_LIFESPAN		50 // 5 seconds
+#define CHAT_MESSAGE_EOL_FADE		10 // 1 second
```

Comment:
For future reference there's a define for SECONDS so you could have just done
`#define CHAT_MESSAGE_EOL_FADE 1 SECONDS`